### ISSUE TITLE
No way to disable smtp_username and password

### ISSUE THREAD

**Issue by halkeye** [lowercasename/gathio#151]
https://github.com/lowercasename/gathio/blob/6f0721686aea7647554ad0f3ee4f6099bff4ef58/src/lib/email.ts#L43-L44

"Nodemailer is configured as the email service, but not all required fields are provided. Please provide all required fields in the config file."

I have a local mail server that proxies out to a service, it doesn't have credentials (maybe it should), but there's no way right now to allow that.

do you want a pr that just changes it to remove those two lines?

**Comment by lowercasename** [comment]
Yes, that would be fantastic! This will also probably solve https://github.com/lowercasename/gathio/issues/148.

**Comment by halkeye** [comment]
Self signed is another issue I have to figure out a plan for. Ill try and submit a pr tonight
